### PR TITLE
Fixing issue #27 Rendering pokemon card to DOM

### PR DESCRIPTION
**Pull request by Mire-web**
Added the function to reveal pokemon on click
Added alt attribute to image tags for Accessibility

### CODE REVIEW

**Comment by JoeDravarol** [comment]
I don't think this should contain the alt attribute part, it should be opened to its own issue.

**Comment by Dex-theDev** [comment]
Also in the js file. i see that you created an array of the back of the cards with [cardBack], but you're iterating through cardFront? Forgive me if that's on purpose, can you explain your code just a tiny bit?

**Comment by Mire-web** [comment]
> Also in the js file. i see that you created an array of the back of the cards with [cardBack], but you're iterating through cardFront? Forgive me if that's on purpose, can you explain your code just a tiny bit?

className/querySelectorAll returns a node list, to access individual elements, it's mapped over, that way the styling or whatever event is effected only on the affected element. So i iterate cardBack to get each card-back to addevent listener to each and then iterate over the cardFront so the change only affects corresponding cardFront. else it would flip all cards if statement is true.

**Comment by Dex-theDev** [comment]
> > Also in the js file. i see that you created an array of the back of the cards with [cardBack], but you're iterating through cardFront? Forgive me if that's on purpose, can you explain your code just a tiny bit?
> 
> className/querySelectorAll returns a node list, to access individual elements, it's mapped over, that way the styling or whatever event is effected only on the affected element. So i iterate cardBack to get each card-back to addevent listener to each and then iterate over the cardFront so the change only affects corresponding cardFront. else it would flip all cards if statement is true.

I would do a forEach for the card array to add the eventlistener

File: app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+let cardBack = document.querySelectorAll('.card-back');
+let cardArray = [...cardBack]
+// REveal pokemon on click
+for(let i=0; i<cardArray.length; i++){
+    
+        cardArray[i].addEventListener('click',()=>{
+            for(let j = 0; j<cardFront.length; j++){
+                if(i == j){
+                    cardFront[j].style.transform = 'none';
```
Comment (JoeDravarol) [inline app.js:12]: As discussed in Discord, this line is causing a visual bug when flipping the card due to not using the `.visible` card provided by @CodingWithDan CSS.

You can look into [ClassList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) to achieve that.

Comment (JoeDravarol) [inline app.js:12]: As for looping the card, as @Dex-theDev suggested, you can look into using [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) method to loop through a nodeList.

```suggestion
let cards = document.querySelectorAll('.card');

cards.forEach(card => {
  // Do something
})
```

**Review by JoeDravarol — CHANGES_REQUESTED** [review]

File: app.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                }
+            }
+        })
+}
```
Comment (JoeDravarol) [inline app.js:16]: The app.js file that is NOT in the public folder should be deleted.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        <div class="card-back card-face">
+            <img class="pokeball" src="poke/poke.png" alt="">
+        </div>
+        <div class="card-front card-face">
+            <img class="weedle dance" src="poke/squirtle.png" alt="">
+        </div>
+    </div>
+<script src="app.js"></script>
+</body>
+</html>
```
Comment (JoeDravarol) [inline index.html:123]: The index.html file that is NOT in the public folder should be deleted.

**Review by JoeDravarol — CHANGES_REQUESTED** [review]
Other than the changes I requested, your code for triggering the animation looks great and work as intended.

**Review by JoeDravarol — APPROVED** [review]

**Comment by MichaelPachec0** [comment]
If we do commit this, make sure we squash this commit instead of committing as is, deleting files like this really messes up git history of files.

**Pull request merged**